### PR TITLE
Change size to eastus2 for FVT

### PR DESCRIPTION
**Pull request by LiliDeng**
NVME pass against azure platform.

### CODE REVIEW

**Comment by LiliDeng** [comment]
@chvalean 
Test run against Azure already done, before fix, the NVME cases abort for Lsv2 size not available in westus2, after this PR, all NVME pass, hyperv cases are still running, but my change won't affect hyperv platform.

https://linuxpipeline.westus2.cloudapp.azure.com/blue/rest/organizations/jenkins/pipelines/upstream-kernel-pipeline/branches/PR-431/runs/2/nodes/208/steps/296/log/?start=0

**Comment by LiliDeng** [comment]
@chvalean Cloud you please merge this? Let know your concern.

**Comment by LiliDeng** [comment]
@ader1990 @chvalean Anyone can review this? Thanks!

File: Jenkinsfile [hunk 1]
```diff
@@ -50,6 +50,7 @@ def prepareEnv(branch, remote, distroVersion, functionalTests, platform) {
       if (functionalTests.contains('FVT')) {
           if (platform == "Azure") {
               env.LISAV2_PARAMS = "-TestCategory 'Functional,Community,Stress,BVT' -TestArea 'KVP,SRIOV,NETWORK,STORAGE,WALA,CORE,KDUMP,LTP,STRESS,BVT,NVME'"
+              LISAV2_AZURE_REGION = "eastus2"
```
Comment (ader1990) [inline Jenkinsfile]: you are changing the region for all the functional tests, please change the commit message.

Comment (LiliDeng) [reply Jenkinsfile]: done, thanks for review

**Pull request merged**